R/run-tests.sh

```diff
@@ -31,8 +31,9 @@ NUM_TEST_WARNING="$(grep -c -e 'Warnings ----------------' $LOGFILE)"
 # Also run the documentation tests for CRAN
 CRAN_CHECK_LOG_FILE=$FWDIR/cran-check.out
 rm -f $CRAN_CHECK_LOG_FILE
-
-NO_TESTS=1 NO_MANUAL=1 $FWDIR/check-cran.sh 2>&1 | tee -a $CRAN_CHECK_LOG_FILE
+# TODO(SPARK-30737) reenable this once packages are correctly installed in Jenkins
+touch $CRAN_CHECK_LOG_FILE
+# NO_TESTS=1 NO_MANUAL=1 $FWDIR/check-cran.sh 2>&1 | tee -a $CRAN_CHECK_LOG_FILE
```

Comment:
This reminds me the following. :)
- 91d990162f ([SPARK-24152][R][TESTS] Disable check-cran from run-tests.sh)

Reply:
You need remove the following line `FAILED=$((PIPESTATUS[0]||$FAILED))` together.

Reply:
Thanks @dongjoon-hyun